<!DOCTYPE html>
<html>
 <!-- Standard Meta -->
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
  <!-- Site Properties -->
  <link rel="stylesheet" href="css/aurora-pack.min.css">
  <link rel="stylesheet" href="css/aurora-theme-base.min.css">
  <link rel="stylesheet" href="css/urku.css">
  <title>ICONUP - INTRODUCTION</title>
  <body class="top">
    <header class="ae-container-fluid ae-container-fluid--full rk-header ">
      <input type="checkbox" id="mobile-menu" class="rk-mobile-menu">
      <label for="mobile-menu">
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
      </label>
      <div class="ae-container-fluid rk-topbar">
        <h1 class="rk-logo"><a href="index.html">ICONUP</a></h1>
        <nav class="rk-navigation">
          <ul class="rk-menu">
            <li class="rk-menu__item"><a href="index.html" class="rk-menu__link">Home</a>
            </li>
            <li class="active rk-menu__item"><a href="introduction.html" class="rk-menu__link">Introduction</a>
            </li>
            <li class="rk-menu__item"><a href="methods.html" class="rk-menu__link">Methods</a>
            </li>
            <li class="rk-menu__item"><a href="#0" class="rk-menu__link">Results and Discussion</a>
              <nav class="rk-menu__sub">
                <ul class="rk-container">
                  <li class="rk-menu__item"><a href="validation.html" class="rk-menu__link">Validation</a></li>
                  <li class="rk-menu__item"><a href="application.html" class="rk-menu__link">Application</a></li>
                </ul>
              </nav>
            </li>
            <li class="rk-menu__item"><a href="conclusion.html" class="rk-menu__link">Conclusion and Acknowledgement</a>
            </li>
            <li class="rk-menu__item"><a href="references.html" class="rk-menu__link">References</a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <section class="ae-container-fluid rk-main">
      <div class="rk-layout-ctrl-mobile">
        <svg viewBox="0 0 9 9" class="layout-mob-1">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-2">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-3">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-4">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
      </div>
      <header class="rk-portfolio-cover  item-inside-1">
        <div class="item-inside__meta">
          <h1 class="ae-u-bolder rk-portfolio-title ">Introduction</h1>
        </div>
      </header>
      <div class="ae-container-fluid ae-container-fluid--inner rk-portfolio--inner">
        <div class="ae-grid ae-grid--collapse au-xs-ptp-1">
			<div class="ae-grid__item item-lg-12 au-lg-ptp-1">
            <p>Metagenomics is the study of microbial communities at the genome level.
			 Their genetic materials are obtained in situ (Bragg and Tyson, 2014). 
			 This direct sequencing of the genetic materials from their own environment
			 is the key advantage of metagenomics because it bypasses the culturing step,
			 which has limited the scope of microorganism research as 99% of microorganisms
			 do not survive in a conventional culture environment (Handelsman, 2004).
			 The advance in sequencing technology has allowed researchers to discover possible
			 pathogens or probiotics for some diseases such as inflammatory bowel disease
			 (Peterson <i>et al.</i>, 2012 and Wang <i>et al.</i>, 2015). This recent success
			 in metagenomics quickly grabbed the interest of many public and private investors
			 (Reardon, 2016).</p>
			<p>Another reason why this field of research became very attractive is that microorganisms thrive everywhere.
			 Specifically in our body, we have on average 70 trillion cells living and 40 trillion cells are microorganisms
			 (Abbott, 2016). Thus, this abundance of microorganisms in our body allowed human microbiome research,
			 which is a community of microorganisms living in association with human cells, to have a wide range of disease
			 applications: cancer, diabetes, infectious disease, and immunology(Greenblum <i>et al.</i>, 2011; Walker <i>et al.</i>, 2015)</p>
          </div>
		<div class="ae-grid ae-grid--collapse">
          <div class="ae-grid__item item-lg-12 au-lg-ptp-1">
            <p>One popular instance of microbiome research is an identification of differentially abundant operational
             taxonomic units (OTUs) on disease cases when compared to healthy controls. Recent studies found a significant
              difference in gut microbiome composition between diabetes patients and healthy controls (Larsen <i>et al.</i>,
               2010 and Qin <i>et al.</i>, 2012).</p>
            <p>However, microorganisms interact with the host and also between themselves,
             and these interactions can give an additional layer of information that the popular approach lacked.
              Thus, research on these interactions can give rise to an interesting and possibly different conclusion. 
              Previous studies on these interactions created association networks using correlation measures where 
              the edges are correlation coefficients or p values to indicate statistically (and potentially biologically)
               significant interactions and the nodes are the microorganisms (Weiss <i>et al.</i>, 2016).
                This approach infers a co-occurrence network for a given microbiome. </p>
 			<p>Here, a novel method for creating association network is presented. This method is a new extension
 			 to an existing differential network inference method (Fairclough <i>et al.</i>, unpublished).
 			  Rather than generating a differential network between two data sets, the new method just uses
 			   one data set and infers a co-occurrence network using proportionality measures and random permutation. 
 			   To validate this methodology, it was compared with several other existing network inference methods
 			    for sensitivity, specificity, and precision. Also, the methodology was successfully applied to a 
 			    case-control study of stroke patients, a data set downloaded from EBI metagenomics database (Yin <i>et al.
 			    </i>, 2015; Mitchell <i>et al.</i>). 
 			    The resulting network successfully retrieved some microorganisms known to be associated with stroke 
 			    and also highlighted some potential interactions. Thus, this methodology can be used as a hypothesis 
 			    generator to detect any (statistically) significantly co-occurring microorganisms in a given data set.</p>
          </div>
          </div>
        </div>

      <div class="ae-container-fluid au-pt-4 au-pb-4">
        <div class="group-buttons"><a href="methods.html" class="au-mt-2 arrow-button arrow-button--right">Next: Methods<span class="arrow-cont">
              <svg>
                <use xlink:href="img/symbols.svg#arrow"></use>
              </svg></span></a></div>
      </div>
    </section>
    <footer class="ae-container-fluid rk-footer ">
      <div class="ae-grid ae-grid--collapse">
        <div class="ae-grid__item item-lg-4 au-xs-ta-center au-lg-ta-left">
          <p class="rk-footer__text rk-footer__contact "> <span class="ae-u-bold">Email: </span><span class="ae-u-bolder"> <a href="#0" class="rk-dark-color ">yk1615@ic.ac.uk </a></span></p>
          <p class="rk-footer__text rk-footer__by"><span class="ae-u-bolder">Yunsoo Kim, June 2016<span></p>
        </div>
      </div>
    </footer>
    <script src="js/svg4everybody.min.js"></script>
    <script>svg4everybody();</script>
  </body>
</html>
