<!DOCTYPE html>
<html>
 <!-- Standard Meta -->
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
  <!-- Site Properties -->
  <link rel="stylesheet" href="css/aurora-pack.min.css">
  <link rel="stylesheet" href="css/aurora-theme-base.min.css">
  <link rel="stylesheet" href="css/urku.css">
  <title>ICONUP - REFERENCES</title>
  <body class="top">
    <header class="ae-container-fluid ae-container-fluid--full rk-header ">
      <input type="checkbox" id="mobile-menu" class="rk-mobile-menu">
      <label for="mobile-menu">
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
        <svg>
          <use xlink:href="img/symbols.svg#bar"></use>
        </svg>
      </label>
      <div class="ae-container-fluid rk-topbar">
        <h1 class="rk-logo"><a href="index.html">ICONUP</a></h1>
        <nav class="rk-navigation">
          <ul class="rk-menu">
            <li class="rk-menu__item"><a href="index.html" class="rk-menu__link">Home</a>
            </li>
            <li class="rk-menu__item"><a href="introduction.html" class="rk-menu__link">Introduction</a>
            </li>
            <li class="rk-menu__item"><a href="methods.html" class="rk-menu__link">Methods</a>
            </li>
            <li class="rk-menu__item"><a href="#0" class="rk-menu__link">Results and Discussion</a>
              <nav class="rk-menu__sub">
                <ul class="rk-container">
                  <li class="rk-menu__item"><a href="validation.html" class="rk-menu__link">Validation</a></li>
                  <li class="rk-menu__item"><a href="application.html" class="rk-menu__link">Application</a></li>
                </ul>
              </nav>
            </li>
            <li class="rk-menu__item"><a href="conclusion.html" class="rk-menu__link">Conclusion and Acknowledgement</a>
            </li>
            <li class="active rk-menu__item"><a href="references.html" class="rk-menu__link">References</a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <section class="ae-container-fluid rk-main">
      <div class="rk-layout-ctrl-mobile">
        <svg viewBox="0 0 9 9" class="layout-mob-1">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-2">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-3">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
        <svg viewBox="0 0 9 9" class="layout-mob-4">
          <rect width="100%" height="100%" fill="currentColor"></rect>
        </svg>
      </div>
      <header class="rk-portfolio-cover  item-inside-1">
        <div class="item-inside__meta">
          <h1 class="ae-u-bolder rk-portfolio-title ">References</h1>
        </div>
      </header>
      	  <div class="ae-container-fluid ae-container-fluid--inner rk-portfolio--inner">
        <div class="ae-grid ae-grid--collapse au-xs-ptp-1">
			<div class="ae-grid__item item-lg-12 au-lg-ptp-1">
<p>Almudevar et al., 2006. Almudevar, A., Klebanov, L. B., Qiu, X., Salzman, P., and Yakovlev, A. Y. (2006). Utility of correlation measures in analysis of gene expression. NeuroRx, 3(3):384–395.</p>
<p>Bastian et al., 2009. Bastian, M., Heymann, S., and Jacomy, M. (2009). Gephi: An open source software for exploring and manipulating networks. Proceedings of the Third Internation ICWSM Conference.</p>
<p>Bastien et al., 2016. Bastien, F., Lamblin, P., Pascanu, R., Bergstra, J., Goodfellow, I. J., Bergeron, A., Bouchard, N., and Bengio, Y. (2016). Using the gpu - theano. http://deeplearning.net/software/theano/tutorial/using gpu.html. Accessed: 20-03-2016.</p>
<p>Benjamini and Hochberg, 1995. Benjamini, Y. and Hochberg, Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society. Series B (Methodological), pages 289–300.</p>
<p>Bergstra et al., 2010. Bergstra, J., Breuleux, O., Bastien, F., Lamblin, P., Pascanu, R., Desjardins, G., Turian, J., Warde-Farley, D., and Bengio, Y. (2010). Theano: A cpu and gpu math compiler in python. In Proc. 9th Python in Science Conf, pages 1–7.</p>
<p>Bikel et al., 2015. Bikel, S., Valdez-Lara, A., Cornejo-Granados, F., Rico, K., Canizales-Quinteros, S., Sober´on, X., Del Pozo-Yauner, L., and Ochoa-Leyva, A. (2015). Combining metagenomics, metatranscriptomics and viromics to explore novel microbial interactions: towards a systems-level understanding of human microbiome. Computational and structural biotechnology journal, 13:390–401.</p>
<p>Bostock, 2015. Bostock, T. (2015). D3.js: Data driven documents. https://d3js.org/, Accessed: 18-03-2016.</p>
<p>Bragg and Tyson, 2014. Bragg, L. and Tyson, G. W. (2014). Metagenomics using next-generation sequencing. Environmental Microbiology: Methods and Protocols, pages 183–201.</p>
<p>Brooks et al., 2015. Brooks, J. P., Edwards, D. J., Harwich, M. D., Rivera, M. C., Fettweis, J. M., Serrano, M. G., Reris, R. A., Sheth, N. U., Huang, B., Girerd, P., et al. (2015). The truth about metagenomics: quantifying and counteracting bias in 16s rrna studies. BMC microbiology, 15(1):66.</p>
<p>Chen et al., 2013. Chen, W., Zhang, C. K., Cheng, Y., Zhang, S., and Zhao, H. (2013). A comparison of methods for clustering 16s rrna sequences into otus. PloS one, 8(8):e70837.</p>
<p>Consortium et al., 2012. Consortium, H. M. P. et al. (2012). Structure, function and diversity of the healthy human microbiome. Nature, 486(7402):207–214.</p>
<p>Fairclough et al., 2016. Fairclough, V., Sim, A., and Pinney, J.W. (2016). Using proportionality measures to analyse changes in inter-genus dependencies within the human microbiota. Unpublished, held at Imperial College London.</p>
<p>Flathemes, 2016. Flathemes (2016). Bootflat. http://bootflat.github.io/index.html. Accessed: 20-03-2016.</p>
<p>Grinberg, 2014. Grinberg, M. (2014). Flask Web Development: Developing Web Applications with Python. ”O’Reilly Media, Inc.”.</p>
<p>Handelsman, 2004. Handelsman, J. (2004). Metagenomics: application of genomics to uncultured microorganisms. Microbiology and molecular biology reviews, 68(4):669–685.</p>
<p>Jacomy and Plique, 2015. Jacomy, A. and Plique, G. (2015). Sigma.js. http://sigmajs.org/, Accessed: 18-03-2016.</p>
<p>Kuczynski et al., 2012. Kuczynski, J., Stombaugh, J., Walters, W. A., Gonz´alez, A., Caporaso, J. G., and Knight, R. (2012). Using qiime to analyze 16s rrna gene sequences from microbial communities. Current protocols in microbiology, pages 1E–5.</p>
<p>Lovell et al., 2015. Lovell, D., Pawlowsky-Glahn, V., Egozcue, J. J., Marguerat, S., and B¨ahler, J. (2015). Proportionality: a valid alternative to correlation for relative data. PLoS Comput Biol, 11(3):e1004075.</p>
<p>McDonald et al., 2012. McDonald, D., Clemente, J. C., Kuczynski, J., Rideout, J. R., Stombaugh, J., Wendel, D., Wilke, A., Huse, S., Hufnagle, J., Meyer, F., et al. (2012). The biological observation matrix (biom) format or: how i learned to stop worrying and love the ome-ome. GigaScience, 1(1):7.</p>
<p>McKinney, 2010. McKinney, W. (2010). Data structures for statistical computing in python. In van der Walt, S. and Millman, J., editors, Proceedings of the 9th Python in Science Conference, pages 51 – 56.</p>
<p>Methvin et al., 2016. Methvin, D., Borchers, K., Sontag, A., Anne-Gaelle, C., Sherov, M., Sexton, A., Smith, A., Schulhof, G., Schiemann, D., Peachey, M., and Resig, J. (2016). jquery: The write less, do more, javascript library. http://https://jquery.com/, Accessed: 18-03-2016.</p>
<p>Morgan et al., 2012. Morgan, X. C., Tickle, T. L., Sokol, H., Gevers, D., Devaney, K. L.,Ward, D. V., Reyes, J. A., Shah, S. A., LeLeiko, N., Snapper, S. B., et al. (2012). Dysfunction of the intestinal microbiome in inflammatory bowel disease and treatment. Genome Biol, 13(9):R79.</p>
<p>Poirot et al., 2003. Poirot, O., O’Toole, E., and Notredame, C. (2003). Tcoffee@ igs: a web server for computing, evaluating and combining multiple sequence alignments. Nucleic acids research, 31(13):3503–3506. </p>
<p>Rajendhran and Gunasekaran, 2011. Rajendhran, J. and Gunasekaran, P. (2011). Microbial phylogeny and diversity: small subunit ribosomal rna sequence analysis and beyond. Microbiological research, 166(2):99–110.</p>
<p>Rocco and Helmers, 2016. Rocco, M. and Helmers, J. H. (2016). Celery: Distributed task queue. http://www.celeryproject.org/. Accessed: 20-03-2016.</p>
<p>Sanfilippo, 2016. Sanfilippo, S. (2016). redis. http://redis.io/. Accessed: 20-03-2016.</p>
<p>Schult and Swart, 2008. Schult, D. A. and Swart, P. (2008). Exploring network structure, dynamics, and function using networkx. In Proceedings of the 7th Python in Science Conferences (SciPy 2008), volume 2008, pages 11–16.</p>
<p>Seabold and Perktold, 2010. Seabold, S. and Perktold, J. (2010). Statsmodels: Econometric and statistical modeling with python. In Proceedings of the 9th Python in Science Conference, pages 57–61.</p>
<p>Sekirov et al., 2010. Sekirov, I., Russell, S. L., Antunes, L. C. M., and Finlay, B. B. (2010). Gut microbiota in health and disease. Physiological Reviews, 90(3):859–904.</p>
<p>Shannon et al., 2003. Shannon, P., Markiel, A., Ozier, O., Baliga, N. S., Wang, J. T., Ramage, D., Amin, N., Schwikowski, B., and Ideker, T. (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome research, 13(11):2498–2504.</p>
<p>Sokol et al., 2016. Sokol, H., Leducq, V., Aschard, H., Pham, H.-P., Jegou, S., Landman, C., Cohen, D., Liguori, G., Bourrier, A., Nion-Larmurier, I., et al. (2016). Fungal microbiota dysbiosis in ibd. Gut, pages gutjnl–2015.</p>
<p>Turnbaugh et al., 2007. Turnbaugh, P. J., Ley, R. E., Hamady, M., Fraser-Liggett, C., Knight, R., and Gordon, J. I. (2007). The human microbiome project: exploring the microbial part of ourselves in a changing world. Nature, 449(7164):804.</p>
<p>Van Der Walt et al., 2011. Van Der Walt, S., Colbert, S. C., and Varoquaux, G. (2011). The numpy array: a structure for efficient numerical computation. Computing in Science & Engineering, 13(2):22–30.</p>
<p>Woese and Fox, 1977. Woese, C. R. and Fox, G. E. (1977). Phylogenetic structure of the prokaryotic domain: the primary kingdoms. Proceedings of the National Academy of Sciences, 74(11):5088–5090.</p>
<p>Zoetendal et al., 2008. Zoetendal, E., Rajili´c-Stojanovi´c, M., and De Vos, W. (2008). High-throughput diversity and functionality analysis of the gastrointestinal tract microbiota. Gut, 57(11):1605–1615.</p>
          </div>


    </section>
    <footer class="ae-container-fluid rk-footer ">
      <div class="ae-grid ae-grid--collapse">
        <div class="ae-grid__item item-lg-4 au-xs-ta-center au-lg-ta-left">
          <p class="rk-footer__text rk-footer__contact "> <span class="ae-u-bold">Email: </span><span class="ae-u-bolder"> <a href="#0" class="rk-dark-color ">yk1615@ic.ac.uk </a></span></p>
          <p class="rk-footer__text rk-footer__by"><span class="ae-u-bolder">Yunsoo Kim, June 2016<span></p>
        </div>
      </div>
    </footer>
    <script src="js/svg4everybody.min.js"></script>
    <script>svg4everybody();</script>
  </body>
</html>
